Representor/HTTP/Adapters/HTTPSirenAdapter.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  SirenAdapter.swift
```

Comment:
I am just curios: why is this file called `HTTPSirenAdapter.swift` and not just `SirenAdapter.swift`? It is in the HTTP folder anyway...